app/forms/decidim/anonymous_codes/admin/code_group_form.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Decidim
+  module AnonymousCodes
+    module Admin
+      class CodeGroupForm < Decidim::Form
+        include TranslatableAttributes
+
+        translatable_attribute :title, String
+        attribute :expires_at, Decidim::Attributes::LocalizedDate
```

Comment:
In the form we are using `datetime_field`, so here we should put
```suggestion
        attribute :expires_at, Decidim::Attributes::TimeWithZone
```

Reply:
Thanks, done.